app/src/features/surveys/telemetry/manage/devices/create/CreateDevicePage.tsx

```diff
@@ -58,7 +58,9 @@ export const CreateDevicePage = () => {
     } catch (error) {
       dialogContext.setErrorDialog({
         dialogTitle: TelemetryDeviceI18N.createErrorTitle,
-        dialogText: TelemetryDeviceI18N.createErrorText,
+        dialogText: !(error as APIError).message.includes('already exists')
```

Comment:
suggestion: This could also be its own function if this check is something that occurs in other places. If not we can just move the text to a constant at the top of this file.